.github/workflows/publish-release.yml

```diff
@@ -61,6 +61,46 @@ jobs:
           fastly_key: ${{ secrets.FASTLY_PURGE_KEY }}
           fastly_service: staging-js-agent.newrelic.com
           asset_path: ${{ join(fromJson(steps.s3-upload.outputs.results).*.Key, ' ') }}
+      - name: Get version number
```

Comment:
Can this be spun off into a separate job in this flow? Publish-release is too important and rarely ran, and rest below (publish-current-to-s3) depends on this job. Just about to cut a release now and would rather not deal with this blocking the pipeline from running, to be selfish.